app/assets/javascripts/components/enroll/new_account_modal.jsx

```diff
@@ -42,6 +42,12 @@ const NewAccountModal = ({ course, passcode, currentUser, closeModal, newAccount
     newAccountInfo = <div><p>{I18n.t('courses.new_account_info')}</p></div>;
   }
 
+  const wikipediaAccountCreation = (
+    <a href={`/users/auth/mediawiki_signup?origin=${window.location}`} className="button auth signup border margin">
+      <i className="icon icon-wiki-logo" />{I18n.t('application.sign_up_extended')}
+    </a>
+  );
+
```

Comment:
This should only be there if the user if not in the instructor role. I think we also need to change to interface messages to make it clear that clicking this is an alternative to requesting an account: the user can either create their own account directly and then sign in and join the program, or they request an account.